website/docs/guidelines/pull-request/index.md

```diff
@@ -0,0 +1,15 @@
+# Pull Request
+
+Pull requests (PRs) are a critical part of the development workflow, enabling collaboration and code review.
+
+Establishing conventions for PRs ensures clarity and consistency, making it easier for team members to understand and review changes.
+
+## General Principles
+
+### Transparency
+
+Building transparency is crucial in the development process. [Opening pull requests in draft mode](https://github.blog/news-insights/product-news/introducing-draft-pull-requests/) is encouraged to solicit early feedback and foster open communication among team members.
```

Comment:
> is encouraged to solicit early feedback and foster open communication among team members.

I do not agree with this point. I often use draft PRs to track my changes, and use CI validation tools while I'm still working on a particular feature. I generally do not seek for feedback in that stage, except when I explicitly ask for it.

Reply:
Opening Draft PR is encouraged to build transparency.
With that, the author can ask early feedback (if needed), otherwise IMHO it can be just a Draft PR used as you said.

The keypoint I wanted to express are:
- be transparent, open Draft PR
- with Draft PR you can ask an early feedback -> you need it? you ask it!

Reply:
@Krusty93 I’ve updated the description; does it seem clearer now?

Reply:
Definitely better and I agree with it, thanks!